nipype/interfaces/dcm2nii.py

```diff
@@ -359,11 +413,7 @@ def _parse_stdout(self, stdout):
                 out_file = None
                 if line.startswith("Convert "):  # output
                     fname = str(re.search('\S+/\S+', line).group(0))
-                    if isdefined(self.inputs.output_dir):
-                        output_dir = self.inputs.output_dir
-                    else:
-                        output_dir = self._gen_filename('output_dir')
-                    out_file = os.path.abspath(os.path.join(output_dir, fname))
+                    out_file = os.path.abspath(fname)
```

Comment:
Do you not want to use `output_dir` at all? It no longer seems to have any effect in this interface.

Reply:
if omitted, `dcm2niix` defaults to save within input folder, so I'd prefer to have it save within the node directory/cwd. The `output_dir` is parsed from `dcm2niix`'s stdout as `fname`

Reply:
Ah, sorry, I misunderstood what was going on here.